bin/main.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:http/http.dart' as http;
+
+Future<String> download(final url) async {
+  final response = await http.get(url);
+  dynamic htmlToParse = '';
```

Comment:
Please avoid using `dynamic`. Instead, you should use `final` (preferred) or `var` (if your variable changes) to let the compiler infer the type, or specify the specific type yourself. If you use `dynamic`, the compiler will not catch type errors for you. Catching type errors early will save you a lot of trouble because they will reveal bugs early.